sound/hybrid_font.h

```diff
@@ -7,10 +7,14 @@ class IgniterConfigFile : public ConfigFile {
     CONFIG_VARIABLE(humStart, 100);
     CONFIG_VARIABLE(volHum, 15);
     CONFIG_VARIABLE(volEff, 16);
+    CONFIG_VARIABLE(clashLockOn, 1);
```

Comment:
This config file was created by NEC, and adding values without clear naming is going to cause problems.
If we're going to add any configuration values to this file, we should probably name them something like "proffieos_clashLockOn" instead of just "clashLockOn".

Reply:
that all sounds very reasonable to me. i think whatever is required for you to have things working that way you want in the main repo is a good thing. also in my experience plecter fans do not want things changed except by plecter. i just didnt want to leave out the real endlock when its a thing long before lockon and lockoff on proffie, it didnt seem appropriate so i took a moment to add support for monophonic endlock. whatever naming conventions the main repo would want to use is ok with me, it has no bearing on my fork. im just trying to be adaptable and participate a little in a foreign world. call it a bucket list thing maybe, to try new things. i hope it helps somehow, im not used to all this and i dont have much time for anything new most days...